frontend/src/metabase/entities/fields.js

```diff
@@ -97,7 +97,9 @@ const Fields = createEntity({
           ),
         );
         // Field values needs to be fetched again once the field is updated metabase#16322
-        await dispatch(Fields.actions.fetchFieldValues(field));
+        await dispatch(
+          Fields.actions.fetchFieldValues(field, { reload: true }),
```

Comment:
This is required to fix the [failed](https://app.circleci.com/jobs/github/metabase/metabase/1621659?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) [tests](https://app.circleci.com/pipelines/github/metabase/metabase/31713/workflows/f0a04b94-200c-4e39-a9a7-5ee5b6a51993/jobs/1621660) for this [backported PR](https://github.com/metabase/metabase/pull/21244)